conda-store-server/tests/user_journeys/utils/api_utils.py

```diff
@@ -136,7 +134,7 @@ def create_token(
         """Create a token with a specified role in a specified namespace."""
         json_data = {
             "primary_namespace": default_namespace,
-            "expiration": time_utils.get_iso8601_time(1),
+            # "expiration": time_utils.get_iso8601_time(1),
```

Comment:
Hmm, I thought we _needed_ an expiration time? If we don't, let's just remove this line.

Reply:
Nope, by default the api will set the expiry to the same expiration as the token that is making the request. 

This line in particular was not generating the expiration in the right format so it was causing an error. By removing this line the generated token will get the default expiration value.